spec/features/guest_registers_for_courses_spec.rb

```diff
@@ -11,7 +11,7 @@
   [x] Registering for courses creates the appropriate course_registrations objects
 } do
 
-  scenario 'guest registers for a class' do
+  pending 'guest registers for a class' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.